pkgs/tools/networking/globalprotect-openconnect/default.nix

```diff
@@ -21,6 +21,12 @@ stdenv.mkDerivation rec {
   patchPhase = ''
     substituteInPlace GPService/gpservice.h \
       --replace /usr/local/bin/openconnect ${openconnect}/bin/openconnect;
+    substituteInPlace GPClient/settingsdialog.ui \
+      --replace /etc/gpservice/gp.conf $out/etc/gpservice/gp.conf;
+    substituteInPlace GPService/gpservice.cpp \
+      --replace /etc/gpservice/gp.conf $out/etc/gpservice/gp.conf;
+    substituteInPlace GPService/CMakeLists.txt \
+      --replace /etc/gpservice $out/etc/gpservice;
```

Comment:
Are those the default configs or the ones being read at runtime?

Reply:
Good question.  It looks like https://github.com/yuezk/GlobalProtect-openconnect/wiki/Configuration was introduced in `1.4.3`.  Probably this is something we should add support for in `nixos/modules/services/networking/globalprotect-vpn.nix`.  But, I don't think I would hold up this version bump for it, esp. since there are security fixes in here.

Reply:
There is a file at `$out/etc/gpservice/gp.conf` with what appears to be sensible default (i.e. no-op) contents, though, FWIW.